ansible/roles/g_nodes/tasks/main.yml

```diff
@@ -10,3 +10,9 @@
   block:
     - ansible.builtin.include_tasks: gpufacts.yml
   tags: gpufacts
+
+- name: Performing Nvidia-DeepLearning frameworks test on the host
```

Comment:
We need `become_user` here set to the new variable (https://docs.ansible.com/ansible/latest/user_guide/become.html#using-become).